### PR TITLE
Create Mapping_in_R.rmd

### PR DESCRIPTION
**Pull request by BrianHJLi**
cc19 mapping in r

### CODE REVIEW

File: Mapping_in_R.rmd [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# Mapping in R
+
+Hanjun Li and Chengchao Jin
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+### Overview
```
Comment (jtr13) [inline Mapping_in_R.rmd]: Please change all the ### headers to ## so you're not skipping from # to ### -- it will look better that way when rendered.

**Review by jtr13 — CHANGES_REQUESTED** [review]

File: Mapping_in_R.rmd (hunk 1 above)
Comment (BrianHJLi) [reply Mapping_in_R.rmd]: Hi Professor, we just changed all ### headers to ##. Please check again, thank you.

**Review by bojum — CHANGES_REQUESTED** [review]
please remove .ds_store

File: Mapping_in_R.rmd (hunk 1 above)
Comment (BrianHJLi) [reply Mapping_in_R.rmd]: Hi, I am also wondering how should I remove .ds_store as I did not remember adding this file.

**Comment by bojum** [comment]
I figured it'd be faster if I deleted it for you.

Thanks for your contribution! The branch has been merged! 🎉

**Pull request merged**